dev/unix/volta-install.sh

```diff
@@ -120,21 +120,19 @@ upgrade_is_ok() {
   return 0
 }
 
-# returns the os name to be used in the packaged release,
-# including the openssl info if necessary
+# returns the os name to be used in the packaged release
 parse_os_info() {
   local uname_str="$1"
-  local openssl_version="$2"
+  local arch="$(uname -m)"
 
   case "$uname_str" in
     Linux)
-      parsed_version="$(parse_openssl_version "$openssl_version")"
-      exit_code="$?"
-      if [ "$exit_code" != 0 ]; then
-        return "$exit_code"
+      if [ "$arch" == "x86_64" ]; then
+        echo "linux"
+      else
+        error "Releases for non x64 architectures are not currently supported."
+        return 1
```

Comment:
Just double-checking—this isn't a regression, is it? I know we had had folks *interested* in doing Arm builds on Linux, and AFAIK we had never *shipped* those but also hadn’t done things which prevented people from doing it themselves… but don't remember if that was the extent of it?

Reply:
Correct, though double-checking we don't actually need that. There is already a `check_architecture` function that prints an error message if the wrong architecture is used.

There was some work to try to get ARM builds for Linux, but I believe it stalled on CI build issues (that I think the approach in this PR should unblock)

Reply:
Hey! Would like to add that I this this *does* cause a regression for arm64 users of Linux trying to build and install volta locally (conversation from https://canary.discord.com/channels/413753499835301908/413753499835301917/1026109337313222736 lower) 😅 
Basically this change prevents `./dev/unix/volta-install.sh --release` from being ran on arm64.

Reply:
Ah, thanks for the call-out @vladfrangu! Will get that reverted (since as noted above it isn’t actually necessary anyway).